jsm.go

```diff
@@ -727,6 +735,78 @@ func (js *js) StreamInfo(stream string, opts ...JSOpt) (*StreamInfo, error) {
 	return resp.StreamInfo, nil
 }
 
+func (js *js) StreamContainedSubjects(stream string, filter ...string) (map[string]uint64, error) {
+	if err := checkStreamName(stream); err != nil {
+		return nil, err
+	}
+
+	if len(filter) > 1 {
```

Comment:
See comment above, if you don't support more than 1, then why make it a `...string`?